api/consumer/searchPostDocumentReference/search_post_document_reference.py

```diff
@@ -45,6 +46,10 @@ def handler(
             expression="subject:identifier",
         )
 
+    config = Config()
+    base_url = f"https://{config.ENVIRONMENT}.api.service.nhs.uk/"
```

Comment:
Do we need to support internal hostnames as well as the public ones? Environments that only have an internal hostname, like dev envs and integ tests etc will likely have invalid link urls as they have no public hostnames.

Reply:
This is true. For now, at least public facing urls will always be valid. We could consider adding some logic to determine whether the internal format needs to be used instead, based on the environment in use, but that will not add any immediate value for users/suppliers at this point so it has been agreed to revisit this after MVP. We might also want to switch to relative URLs, which would be valid per FHIR spec since the link's .url is a "uri" type: https://hl7.org/fhir/R4/datatypes.html#uri